.github/workflows/deploy_production.yml

```diff
@@ -0,0 +1,54 @@
+name: Deploy production app to Firebase App Distribution
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  deploy_android:
+    name: Deploy Android app to Firebase App Distribution
+    runs-on: ubuntu-latest
+    timeout-minutes: 30
+    defaults:
+      run:
+        working-directory: ./sample
+    steps:
+      - name: Set up JDK
+        uses: actions/setup-java@v2
+        with:
+          distribution: 'temurin'
+          java-version: '17'
+
+      - name: Checkout source code
+        uses: actions/checkout@v2.3.2
+
+      - name: Cache Gradle
+        uses: actions/cache@v2
+        with:
+          path: |
+            ~/.gradle/caches/modules-*
+            ~/.gradle/caches/jars-*
+            ~/.gradle/caches/build-cache-*
+          key: ${{ runner.os }}-gradle-${{ hashFiles('**/*.gradle*') }}
+          restore-keys: |
+            ${{ runner.os }}-gradle-
+
+      - name: Decode singing properties
+        run: echo ${{ secrets.ANDROID_SIGNING_PROPERTIES }} | base64 --decode > signing.properties
+
+      - name: Decode release keystore
+        run: echo "${{ secrets.ANDROID_RELEASE_KEYSTORE }}" | base64 --decode > config/release.keystore
```

Comment:
`ANDROID_SIGNING_PROPERTIES` and `ANDROID_RELEASE_KEYSTORE` need to be moved into `production` env

Reply:
@luongvo I don't think so. As in `env`, we can see the file content that is not safe. I think to persist them in `Secrets` is more suitable. 🙏🏻

Reply:
answered you [here](https://nimble-co.slack.com/archives/C041K5U0D8R/p1700816884995419?thread_ts=1700816697.732369&cid=C041K5U0D8R) 🤗